🐙 source-declarative-manifest: run up-to-date pipeline [2024-11-25]

airbyteio opened the pull request:
# Update source-declarative-manifest

This PR was autogenerated by running `airbyte-ci connectors --name=source-declarative-manifest up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- Update versions of libraries in poetry.: Successful

- PATCH bump source-declarative-manifest version to 6.5.3: Successful

- Build source-declarative-manifest docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Successful

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful




## Dependency updates

We use [`syft`](https://github.com/anchore/syft) to generate a SBOM for the latest connector version and the one from the PR.
It allows us to spot the dependencies that have been updated at all levels and for all types of dependencies (system, python, java etc.).
Here are the dependencies that have been updated compared to `airbyte/source-declarative-manifest:latest`.
Keep in mind that `:latest` does not always match the connector code on the main branch. 
It is the latest released connector image when the head commit of this branch was created.

| Type | Name | State | Previous Version | New Version |
|------|------|-------|-------------|------------------|
| python | airbyte-cdk | updated | 6.6.7 | **6.5.2** |
| python | attrs | updated | 23.2.0 | **24.2.0** |
| deb | base-files | updated | 12.4+deb12u6 | **12.4+deb12u8** |
| deb | bsdutils | updated | 1:2.38.1-5+deb12u1 | **1:2.38.1-5+deb12u2** |
| python | cffi | updated | 1.16.0 | **1.17.1** |
| python | cryptography | updated | 42.0.8 | **43.0.3** |
| python | distlib | updated | 0.3.8 | **0.3.9** |
| python | filelock | updated | 3.15.4 | **3.16.1** |
| python | genson | updated | 1.3.0 | **1.2.2** |
| python | langsmith | updated | 0.1.143 | **0.1.145** |
| deb | libarchive13 | updated | 3.6.2-1+deb12u1 | **3.6.2-1+deb12u2** |
| deb | libblkid1 | updated | 2.38.1-5+deb12u1 | **2.38.1-5+deb12u2** |
| deb | libc-bin | updated | 2.36-9+deb12u7 | **2.36-9+deb12u9** |
| deb | libc6 | updated | 2.36-9+deb12u7 | **2.36-9+deb12u9** |
| deb | libcurl4 | updated | 7.88.1-10+deb12u6 | **7.88.1-10+deb12u8** |
| deb | libexpat1 | updated | 2.5.0-1 | **2.5.0-1+deb12u1** |
| deb | libglib2.0-0 | updated | 2.74.6-2+deb12u3 | **2.74.6-2+deb12u4** |
| deb | libglib2.0-data | updated | 2.74.6-2+deb12u3 | **2.74.6-2+deb12u4** |
| deb | libmount1 | updated | 2.38.1-5+deb12u1 | **2.38.1-5+deb12u2** |
| deb | libnghttp2-14 | updated | 1.52.0-1+deb12u1 | **1.52.0-1+deb12u2** |
| deb | libnss3 | updated | 2:3.87.1-1 | **2:3.87.1-1+deb12u1** |
| deb | libsmartcols1 | updated | 2.38.1-5+deb12u1 | **2.38.1-5+deb12u2** |
| deb | libsqlite3-0 | updated | 3.40.1-2 | **3.40.1-2+deb12u1** |
| deb | libssl3 | updated | 3.0.13-1~deb12u1 | **3.0.15-1~deb12u1** |
| deb | libsystemd0 | updated | 252.26-1~deb12u2 | **252.31-1~deb12u1** |
| deb | libudev1 | updated | 252.26-1~deb12u2 | **252.31-1~deb12u1** |
| deb | libuuid1 | updated | 2.38.1-5+deb12u1 | **2.38.1-5+deb12u2** |
| python | more-itertools | updated | 10.3.0 | **10.5.0** |
| deb | mount | updated | 2.38.1-5+deb12u1 | **2.38.1-5+deb12u2** |
| python | msgpack | updated | 1.0.8 | **1.1.0** |
| python | nltk | updated | 3.9.1 | **3.8.1** |
| deb | openssl | updated | 3.0.13-1~deb12u1 | **3.0.15-1~deb12u1** |
| python | orjson | updated | 3.10.11 | **3.10.12** |
| python | packaging | updated | 23.2 | **24.2** |
| python | pkginfo | updated | 1.11.1 | **1.11.2** |
| python | platformdirs | updated | 3.11.0 | **4.3.6** |
| python | pydantic | updated | 2.10.0 | **2.10.1** |
| python | pydantic_core | updated | 2.27.0 | **2.27.1** |
| python | pyproject_hooks | updated | 1.1.0 | **1.2.0** |
| python | tomli | updated | 2.0.1 | **2.1.0** |
| python | tomlkit | updated | 0.12.5 | **0.13.2** |
| python | tqdm | updated | 4.67.0 | **4.67.1** |
| python | trove-classifiers | updated | 2024.7.2 | **2024.10.21.16** |
| python | urllib3 | updated | 2.2.2 | **2.2.3** |
| deb | util-linux | updated | 2.38.1-5+deb12u1 | **2.38.1-5+deb12u2** |
| deb | util-linux-extra | updated | 2.38.1-5+deb12u1 | **2.38.1-5+deb12u2** |
| python | virtualenv | updated | 20.26.3 | **20.27.1** |
| python | wcmatch | updated | 10.0 | **8.4** |
| python | wheel | updated | 0.43.0 | **0.44.0** |
| python | wrapt | updated | 1.16.0 | **1.17.0** |
| python | autocommand | added | not present | **2.2.2** |
| python | backports.tarfile | added | not present | **1.2.0** |
| python | importlib_metadata | added | not present | **8.5.0** |
| python | inflect | added | not present | **7.3.1** |
| python | jaraco.collections | added | not present | **5.1.0** |
| python | jaraco.context | added | not present | **5.3.0** |
| python | jaraco.functools | added | not present | **4.0.1** |
| python | jaraco.text | added | not present | **3.12.1** |
| python | my-test-package | added | not present | **1.0** |
| python | setuptools | added | not present | **75.6.0** |
| python | typeguard | added | not present | **4.3.0** |
| python | zipp | added | not present | **3.21.0** |
| python | dunamai | removed | 1.23.0 | **not present** |
| python | python-ulid | removed | 3.0.0 | **not present** |

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=48666) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=48666) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=48666) it.</sub>

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
Update versions of libraries in poetry.
PATCH bump source-declarative-manifest version to 6.5.3
Build source-declarative-manifest docker image for platform(s) linux/amd64, linux/arm64
Get dependency updates
Create or update pull request on Airbyte repository
Add changelog entry

cgardens commented:
We are closing this PR due to inactivity. We regularly close PRs that were created more than 6 months ago. If you'd like to keep working on it, please feel free reopen it. We'd be happy to keep working with you.
